public/docs/ts/latest/guide/user-input.jade

```diff
@@ -86,58 +89,53 @@ figure.image-display
   and the component does nothing.
 
-  Type in the input box, and watch the display update with each keystroke. *Voila!*
+  Type something in the input box, and watch the display update with each keystroke. *Voila!*
 
 figure.image-display
     img(src='/resources/images/devguide/user-input/keyup-loop-back-anim.gif' alt="loop back")
 .l-sub-section
   :marked
-    **This won't work at all unless we bind to an event**.
+    **This won't work unless you bind to an event**.
 
     Angular only updates the bindings (and therefore the screen)
-    if we do something in response to asynchronous events such as keystrokes.
+    if you do something in response to asynchronous events, such as keystrokes.
 
-    That's why we bind the `keyup` event to a statement that does ... well, nothing.
-    We're binding to the number 0, the shortest statement we can think of.
-    That is all it takes to keep Angular happy. We said it would be clever!
+    By binding the `keyup` event
+    to the number 0, the shortest statement we can think of, the statement
+    does nothing, and Angular is happy.
 :marked
-  That template reference variable is intriguing. It's clearly easier to get to the textbox with that
-  variable than to go through the `$event` object. Maybe we can rewrite our previous
-  keyup example so that it uses the variable to get the user's input. Let's give it a try.
+  It's easier to get to the textbox with the template reference
+  variable than to go through the `$event` object. Here's a rewrite of the previous
+  keyup example so that it uses the variable to get the user's input.
 +makeExample('user-input/ts/app/keyup.components.ts', 'key-up-component-2' ,'app/keyup.components.ts (v2)')(format=".")
 :marked
-  That sure seems easier.
-  An especially nice aspect of this approach is that our component code gets clean data values from the view.
+  An especially nice aspect of this approach is that the component code gets clean data values from the view.
   It no longer requires knowledge of the `$event` and its structure.
 
 <a id="key-event"></a>
 .l-main-section
 :marked
   ## Key event filtering (with `key.enter`)
-  Perhaps we don't care about every keystroke.
-  Maybe we're only interested in the input box value when the user presses Enter, and we'd like to ignore all other keys.
-  When we bind to the `(keyup)` event, our event handling statement hears *every keystroke*.
-  We could filter the keys first, examining every `$event.keyCode`, and update the `values` property only if the key is Enter.
+  When you bind to the `(keyup)` event, the event handling statement hears *every keystroke*.
 
-  Angular can filter the key events for us. Angular has a special syntax for keyboard events.
-  We can listen for just the Enter key by binding to Angular's `keyup.enter` pseudo-event.
+  However, you can listen for just the Enter key by binding to Angular's `keyup.enter` pseudo-event.
```

Comment:
Agreed that the reason for the keyup.enter alternative should be in there (as succinctly as possible, while still being clear).